docs/src/DeveloperGuide/LogicSeparation.md

```diff
@@ -0,0 +1,493 @@
+# QAT Application: Logic separation
+
+## Architecture considerations
+
+In this section, we will discuss some classical architectures and then make
+considerations regarding quantum architectures.
+
+### Motivation: Classical co-processors
+
+Before diving into quantum architectures, we first make a brief note on
+classical architectures: For decades we have seen classical architectures
+composed of multiple processing units. For instance, in the nineties it was
+common to have a co-processor supporting the main x86 architecture with such
+tasks as floating point operations. Other co-processing architectures were built
+to enhance the computers sound capabilities and in 2001, we saw graphical
+processing units (GPUs) emerge. To this day, GPUs are still an essentially
+co-processor for many computing tasks including gaming, physics simulation and
```

Comment:
```suggestion
co-processor for many computing tasks including gaming, physics simulation, and
```
I'm a big fan of the Oxford comma. I'm hoping you will be too 😊.

Reply:
Though it _does_ seem like you've consistently not used Oxford commas throughout the text, so maybe it is your stylistic choice.

Reply:
It is not something I have ever used, however, I am not very picky with regard to stylistic choices for written language, so I am happy to accept these changes :)